rbac/common/task/task_address.py

```diff
@@ -13,9 +13,13 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 """Addresses and accesses task objects on the blockchain"""
+# pylint: disable=unused-import
+
 from rbac.common import addresser
 from rbac.common.base.base_address import AddressBase
 from rbac.common.base.base_relationship import BaseRelationship
+from rbac.common.protobuf import task_state_pb2
```

Comment:
what's the logic behind including these and disabling the pylint check? are there side effects you need from import?

Reply:
The base class discovers the protobufs using the name of the object_type. However, they still must be loaded.